packages/itwinui-css/src/header/header-buttons.scss

```diff
@@ -248,15 +248,15 @@
 }
 
 @mixin iui-header-breadcrumb-button-icon {
```

Comment:
Can the `iui-icon-style` mixin be used for the icons in `header-buttons.scss`?

Reply:
Yes, thank you. I also changed the width/height in select.scss.